smartsim/_core/launcher/step/lsfStep.py

```diff
@@ -163,6 +177,9 @@ def get_launch_cmd(self) -> t.List[str]:
         :rtype: list[str]
         """
         jsrun = self.run_settings.run_command
+        if not jsrun:
+            logger.warning("jsrun not found in PATH")
+            raise RuntimeError("Could not find jsrun in PATH")
```

Comment:
Can we do this with a RunSettings (e.g. local on slurm)?

Reply:
Do _this_ what?

Reply:
@al-rigazzi Can we assume that a failure to find & expand jsrun is worth raising an exception that will stop the experiment?